content/application/transparency-dev-summit.html

```diff
@@ -0,0 +1,46 @@
+---
+url: /transparency-dev-summit
```

Comment:
"transparency" and "dev" are duplicated in the actual URL (i.e. https://transparency.dev/transparency-dev-summit).

Reply:
Fixed